hatchways/team-athens
# Feature/show product list

### What this PR does (required):
- show products inside a list and remove each

### Screenshots / Videos (required):
![Screenshot 2021-06-25 070724](https://user-images.githubusercontent.com/13183518/123415992-05860280-d584-11eb-862f-216d3d94b899.png)

### Any information needed to test this feature (required):
- login
- click one of the lists
- populate lists to see them update
- delete from a list to see it update
